src/Header/SkipLink/SkipLink.jsx

```diff
@@ -42,29 +42,22 @@ export default class SkipLinks extends Component {
 
 	render() {
 		return (
-			<ul className={styles.list} aria-label="Accessibility links">
-				<li>
-					<a
-						href={`#${this.props.skipLinkId}`}
-						className={styles.link}
-						onClick={this.handleClick}
-					>
-						Skip to content
-					</a>
-				</li>
-				<li>
-					<a
-						href="https://www.nice.org.uk/accessibility"
-						className={styles.link}
-					>
-						Accessibility help
-					</a>
-				</li>
-			</ul>
+			<a
+				href={this.props.to}
+				className={styles.link}
+				onClick={this.handleClick}
+			>
+				{this.props.children}
+			</a>
 		);
 	}
 }
 
-SkipLinks.propTypes = {
-	skipLinkId: PropTypes.string.isRequired,
+SkipLink.propTypes = {
+	to: PropTypes.string,
```

Comment:
Just small thing here - I think the `to` prop is required, and `label` isn't needed